openapis/swagger.yaml

```diff
@@ -1350,6 +1381,18 @@ definitions:
       rev:
         type: string
         pattern: '^[0-9a-f]{8}$'
+  OutputRefWithURL:
```

Comment:
what's this?

Reply:
Good catch, it's an orphan object schema that I made and ended up not needing. I'll delete it.